symplyphysics/laws/relativistic/vector/relativistic_velocity_orthogonal_to_movement.py

```diff
@@ -45,6 +45,8 @@ def orthogonal_velocity_component_in_lab_frame_law(
         lorentz_factor_def.velocity: vector_magnitude(proper_frame_velocity_),
     })
 
+    # Note that `dot(v, u_parallel') = dot(v, u)`, since `dot(v, u_orthogonal) = 0` and the dot product is linear in
```

Comment:
You wrote a note but did not project as I suggested. Now your law does not correspond to description.

Reply:
Fixed